pkg/registry/flowcontrol/rest/storage_flowcontrol.go

```diff
@@ -71,3 +85,80 @@ func (p RESTStorageProvider) v1alpha1Storage(apiResourceConfigSource serverstora
 func (p RESTStorageProvider) GroupName() string {
 	return flowcontrol.GroupName
 }
+
+func (p RESTStorageProvider) PostStartHook() (string, genericapiserver.PostStartHookFunc, error) {
+	return PostStartHookName, func(hookContext genericapiserver.PostStartHookContext) error {
+		flowcontrolClientSet := flowcontrolclient.NewForConfigOrDie(hookContext.LoopbackClientConfig)
+		go wait.PollImmediateUntil(time.Minute, func() (bool, error) {
```

Comment:
I think we can use `wait.Forever` instead of `wait.PollImmediate`.

Reply:
`Forever` impose an initial delay on the first try, and also we should honor the stop channel provided by `PostStartHookContext`.